docs/CMakeLists.txt

```diff
@@ -1,4 +1,11 @@
-add_custom_target(doc)
+if(LCM_ENABLE_DOC)
```

Comment:
did you mean to define `option(LCM_ENABLE_DOC "..." ON)` somewhere? I don't see it in this PR